### PR TITLE
angle-grinder 0.7.3 (new formula)

### PR DESCRIPTION
**Pull request by rcoh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by rcoh** [comment]
Whoops, left trailing whitespace when fixing spacing to be consistent with other formulas.

File: Formula/angle-grinder.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix
+  end
+
+  test do
+    (testpath/"logs.txt").write("{\"key\": 5}")
+    system "#{bin}/agrind", "--file", "logs.txt", "* | json | count by key"
```
Comment (ilovezfs) [inline Formula/angle-grinder.rb]: Please `assert_equal` or `assert_match` something here rather than testing only for a zero exit code.

**Review by ilovezfs — DISMISSED** [review]

File: Formula/angle-grinder.rb (hunk 1 above)
Comment (rcoh) [reply Formula/angle-grinder.rb]: Done, thanks.

**Comment by rcoh** [comment]
@ilovezfs changes addressed, let me know if there's anything else needed on my side.

File: Formula/angle-grinder.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix
+  end
+
+  test do
+    (testpath/"logs.txt").write("{\"key\": 5}")
+    assert_match "_count", shell_output("#{bin}/agrind --file logs.txt '* | json | count by key'")
```
Comment (commitay) [inline Formula/angle-grinder.rb]: Can we match output from the file rather than the command itself?

**Comment by rcoh** [comment]
Sure @commitay, updated.

**Review by commitay — APPROVED** [review]

**Comment by commitay** [comment]
Thanks for your first contribution to Homebrew @rcoh!

**Pull request closed without merging**